boofuzz/connections/netconf_connection.py

```diff
@@ -0,0 +1,77 @@
+import warnings
+
+from boofuzz.connections import itarget_connection
+
+
+class NETCONFConnection(itarget_connection.ITargetConnection):
+    """
+    ITargetConnection implementation for NETCONF server connections.
+
+    Args:
+        host (str): IP address of NETCONF server.
+        port (int): port of NETCONF server.
+        username (str): NETCONF server login username.
+        password (str): NETCONF server login password.
+        datastore (str): NETCONF server datastore to send fuzz requests to.
+        hostkey_verify(bool): Verify SSH host key when connecting to NETCONF server
+    """
+
+    def __init__(self, host, port, username, password, datastore, hostkey_verify):
+        self.host = host
+        self.port = port
+        self.username = username
+        self.password = password
+        self.datastore = datastore
+        self.hostkey_verify = hostkey_verify
+        self._received_data = None
+        self._conn = None
+
+    def open(self):
+        try:
+            from ncclient import manager  # pytype: disable=import-error
+        except ImportError:
+            warnings.warn("Importing ncclient package failed. Please install it using pip.", UserWarning, stacklevel=2)
```

Comment:
`stacklevel=2` is not needed here, we want the warning to be emitted directly from this function.
I can't remember why we used that for crc32c.